Assets/Scripts/RedRunner/Utilities/CameraController.cs

```diff
@@ -78,8 +78,18 @@ void Update ()
 			}
 		}
 
+        public void Follow(Transform followee)
```

Comment:
Nit: tabs/spaces mismatch

Reply:
Oh gross, does this project use tabs? I have mine set to 4 spaces.